### PR TITLE
[Term Entry] UI/UX Focus group

### LINKED ISSUE #3889: [Term Entry] UI and UX Design UIUX (User Research) Focus Groups
### Reason/inspiration (optional)

We would like a new term entry on the `focus-group` term for [UIUX](https://www.codecademy.com/resources/docs/uiux): Focus group. The entry should go in a new file under `docs/content/uiux/concepts/focus-group/focus-group.md`.

The entry should describe the term and how it is used. Be sure to include images to aid understanding, if required.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ShatilKhan**
### Description

Closes #3889 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/uiux/focus-group/focus-group.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - 'Methods'
+  - 'User Research'
+  - 'Design Thinking'
+CatalogContent:
+  - 'learn-ux-design'
+  - 'paths/human-computer-interaction'
+---
+
+## UI/UX Focus Group
+
```
Comment (dakshdeepHERE) [inline content/uiux/focus-group/focus-group.md:18]:
```suggestion
```
No need for this header

File: content/uiux/focus-group/focus-group.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+Title: 'UI/UX Focus Group'
+Description: 'A comprehensive guide to conducting and leveraging UI/UX focus groups for user-centered design.'
+Subjects:
+  - 'Human-Computer Interaction'
+  - 'User Experience Design'
+  - 'User Interface Design'
```
Comment (dakshdeepHERE) [inline content/uiux/focus-group/focus-group.md:7]: This is wrong choose subjects only from this [file](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md)

File: content/uiux/focus-group/focus-group.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Tags:
+  - 'Methods'
+  - 'User Research'
```
Comment (dakshdeepHERE) [inline content/uiux/focus-group/focus-group.md:10]:
```suggestion
  - 'UI'
```

File: content/uiux/focus-group/focus-group.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  - 'Design Thinking'
```
Comment (dakshdeepHERE) [inline content/uiux/focus-group/focus-group.md:11]:
```suggestion
  - 'UX'
```

File: content/uiux/focus-group/focus-group.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+## UI/UX Focus Group
+
+The **UI/UX Focus Group** is a powerful method in the realm of Human-Computer Interaction (HCI) and User Experience (UX) design. It serves as a means to gather valuable insights from potential users, allowing designers and developers to create more user-centric and intuitive products.
+
+## Purpose
+
+UI/UX focus groups are designed to:
+
```
Comment (dakshdeepHERE) [inline content/uiux/focus-group/focus-group.md:24]:
```suggestion
```

File: content/uiux/focus-group/focus-group.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+- **Understand User Perspectives:** Gain a deeper understanding of how users interact with a product or service.
+- **Collect Feedback:** Obtain direct feedback on design elements, usability, and overall user experience.
+- **Identify Pain Points:** Identify pain points and challenges users may face during interaction.
+- **Generate Ideas:** Stimulate creative thinking and generate new design ideas.
+- **Validate Design Decisions:** Validate design decisions through user opinions and preferences.
```
Comment (dakshdeepHERE) [inline content/uiux/focus-group/focus-group.md:29]:
```suggestion
- **Understand User Perspectives:** Gain a deeper understanding of how users interact with a product or service.

- **Collect Feedback:** Obtain direct feedback on design elements, usability, and overall user experience.

- **Identify Pain Points:** Identify pain points and challenges users may face during interaction.

- **Generate Ideas:** Stimulate creative thinking and generate new design ideas.

- **Validate Design Decisions:** Validate design decisions through user opinions and preferences.
```
Let's be consistent with the spacing throughout the entry

File: content/uiux/focus-group/focus-group.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+## Structure
+
+A typical UI/UX focus group involves:
```
Comment (dakshdeepHERE) [inline content/uiux/focus-group/focus-group.md:33]:
```suggestion
A typical **UI/UX focus group** involves:
```

File: content/uiux/focus-group/focus-group.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+1. **Selection of Participants:** Carefully select a diverse group of participants representing the target user demographic.
+  
+2. **Preparation of Materials:** Prepare prototypes, wireframes, or design concepts for participants to evaluate.
+
+3. **Facilitation:** A skilled facilitator guides the discussion, encouraging participants to express their thoughts, opinions, and concerns.
+
+4. **Observation:** Designers and stakeholders observe the session, taking notes on participant reactions and comments.
+
+5. **Post-Session Analysis:** Analyze the collected data to derive actionable insights for design improvements.
```
Comment (dakshdeepHERE) [inline content/uiux/focus-group/focus-group.md:43]:
```suggestion
- **Selection of Participants:** Carefully select a diverse group of participants representing the target user demographic.
  
- **Preparation of Materials:** Prepare prototypes, wireframes, or design concepts for participants to evaluate.

- **Facilitation:** A skilled facilitator guides the discussion, encouraging participants to express their thoughts, opinions, and concerns.

- **Observation:** Designers and stakeholders observe the session, taking notes on participant reactions and comments.

- **Post-Session Analysis:** Analyze the collected data to derive actionable insights for design improvements.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
For now work on these comments @ShatilKhan.

**Comment by dakshdeepHERE** [comment]
@ShatilKhan can you commit the changes? It's been over a week now.

**Comment by dakshdeepHERE** [comment]
Hey @ShatilKhan, can you please work on the comments above please so we can move forward with the PR? We will wait another week and then close with PR moving forward with someone else working on this PR.

**Comment by ShatilKhan** [comment]
Yeah sorry for the delay, I'll work on them right away

**Comment by dakshdeepHERE** [comment]
@ShatilKhan I'm looking forward to the changes.

**Review by ShatilKhan — COMMENTED** [review]
Done with the proposed changes

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review @ShatilKhan :)

File: content/uiux/focus-group/focus-group.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+- **Selection of Participants:** Carefully select a diverse group of participants representing the target user demographic.
+  
+- **Preparation of Materials:** Prepare prototypes, wireframes, or design concepts for participants to evaluate.
+
+- **Facilitation:** A skilled facilitator guides the discussion, encouraging participants to express their thoughts, opinions, and concerns.
+
+- **Observation:** Designers and stakeholders observe the session, taking notes on participant reactions and comments.
+
+- **Post-Session Analysis:** Analyze the collected data to derive actionable insights for design improvements.
```
Comment (ishg-153) [inline content/uiux/focus-group/focus-group.md:41]: Move the structure before the purpose.

File: content/uiux/focus-group/focus-group.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+1. **Introduction:** Brief participants on the purpose of the session and introduce the app redesign.
+
+2. **Task Scenarios:** Participants are given scenarios (e.g., transferring money, checking account balance) to perform using the redesigned app.
+
+3. **Feedback Gathering:** Participants provide feedback on the app's usability, visual appeal, and overall experience.
+
+4. **Group Discussion:** Facilitate a discussion on common themes, preferences, and pain points identified during the tasks.
+
+5. **Closing Remarks:** Thank participants, provide incentives if applicable, and communicate next steps in the design process.
```
Comment (ishg-153) [inline content/uiux/focus-group/focus-group.md:67]: The example seems very similar to `## Structure`. Try to explain what focus group interactions would be like through it.

File: content/uiux/focus-group/focus-group.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+
+## Conclusion
+
+UI/UX focus groups play a crucial role in creating user-centered designs. By understanding user perspectives and incorporating direct feedback, designers can refine and optimize products to meet the needs and expectations of their target audience.
```
Comment (ishg-153) [inline content/uiux/focus-group/focus-group.md:71]: This seems unnecessary and isn't accurate either. Please remove this.

**Comment by ishg-153** [comment]
@ShatilKhan You've made one of the suggested changes. Can you address the remaining changes as well?

**Review by ishg-153 — APPROVED** [review]
@ShatilKhan I went ahead and removed the example from the entry and renamed the subheadings. This is now ready to merge.

I apologize for the delay on my part.

**Pull request merged**